08-readmes.Rmd

```diff
@@ -15,11 +15,11 @@ ottrpal::include_slide("https://docs.google.com/presentation/d/1LMurysUhCjZb7DVF
 
 Documentation is an important but sometimes overlooked part of creating a reproducible analysis! There are two parts of documentation we will discuss here: 1) In notebook descriptions and 2) READMEs
 
-Both these notebook descriptions and READMEs are written in markdown -- a shorthand for html (the same as the documentation parts of your code). If you aren't familiar, markdown is such a handy tool and we encourage you to learn it (it doesn't take too long), here's a [quick guide](https://www.markdownguide.org/cheat-sheet/) to get you started.
```

Comment:
I don't know that it is technically correct to call markdown a shorthand for html. It wasn't clear to me what point we are trying to make with the text in the parentheses. I tried to make this sentence more clear...